.github/workflows/verify-latest-deps.yaml

```diff
@@ -0,0 +1,39 @@
+# ********************************************************************************
+#  Copyright (c) 2024 Contributors to the Eclipse Foundation
```

Comment:
2025?

Reply:
The copyright header always contains the year _of inception_, i.e. the year that the file has been created originally. All subsequent changes can be derived from the git log, according to the Eclipse Foundation ...